emailer-ses.js

```diff
@@ -11,10 +11,18 @@ Emailer.init = function(params, callback){
     }
     
     meta.settings.get('emailer-ses', function(err, settings) {
-        if(!err && settings && settings.accessKeyID && settings.secretAccessKey && settings.region && settings.fromAddress) {
-            var creds = new aws.Credentials(settings.accessKeyID, settings.secretAccessKey, null);
-            ses = new aws.SES({credentials: creds, region: settings.region, apiVersion: '2010-12-01'});
-            fromAddress = settings.fromAddress;
+        if(!err && settings && settings.region && settings.fromAddress) {
+            var creds,
+                iam = new aws.IAM();
```

Comment:
Looks like it was left in by error. I attempted to use to to check for an iam user but there is no way to.